src/components/Header.vue

```diff
@@ -103,92 +90,118 @@ export default {
 
       return 0;
     },
-    randomPoster () {
+    randomBanner () {
       const rand = Math.floor(Math.random() * this.$store.getters.allMoviesAsArray.length);
 
       if (this.$store.getters.allMoviesAsArray[rand]) {
-        return `https://image.tmdb.org/t/p/original${this.$store.getters.allMoviesAsArray[rand].movie.poster_path}`;
+        return `https://image.tmdb.org/t/p/original${this.$store.getters.allMoviesAsArray[rand].movie.backdrop_path}`;
       } else {
-        return "https://live.staticflickr.com/65535/47980547206_7a02cb7f3f_h.jpg";
+        return "https://image.tmdb.org/t/p/original/1TvNazsE9WvRIxxeZkvL7IVVgzD.jpg";
```

Comment:
guessing this doesn't scale with users D:

Reply:
Sure it does. That's the banner from Jurassic Park. That's universal.